dev_guide/builds/triggering_builds.adoc

```diff
@@ -405,3 +412,26 @@ Configuration change triggers currently only work when creating a new
 `BuildConfig`. In a future release, configuration change triggers will also be
 able to launch a build whenever a `BuildConfig` is updated.
 ====
+
+[[setting-triggers]]
+=== Setting Webhook Triggers Manually
+
+Triggers on configurations that are not created automatically can be generated
+with `oc set triggers`. For example, to set a GitHub webhook trigger on a Build
+configuration, use:
+
+`oc set triggers bc <name> --from-github`
+
+To set an imagechange trigger, use
+
+`oc set triggers bc <name> --from-image='<image>'` 
+
+[NOTE]
+====
+When a webhook trigger already exists, the respective flags will regenerate the
+webhook secret.
```

Comment:
"When a webhook trigger already exists, adding it again will regenerate the webhook secret"